bin/phonegap.js

```diff
@@ -5,16 +5,42 @@
  */
 
 var CLI = require('../lib/cli');
+var cli = new CLI();
+var tracking = cli.tracking;
+var version = require('../package.json').version;
+
+if(tracking.statusUnknown()) {
+    tracking.prompt(function(){
+        // special case, if it is a 'phonegap tracking' command we don't want
+        // to run it because the prompt will have already done the work
+        if(process.argv[2] != "tracking"){
+            runPhoneGapCommand();
+        }
+    });
+}
+else {
+    runPhoneGapCommand();
+}
 
 /*!
  * Run the command-line client.
  */
+function runPhoneGapCommand() {
+
+    cli.argv(process.argv, function(e) {
+
+        // tracking module will skip if it is not enabled
+
+        tracking.trackEvent("pg-cli " + version,
```

Comment:
I'd suggest using `phonegap@version` instead of `pg-cli version` because it matches the npm name and version format.